src/lib/components/auth-cards/ForgetPWCard.svelte

```diff
@@ -15,16 +15,18 @@
   export let height;
   export let store;
   export let sendUserInfo;
+  export let test;
 
-  let btnDisabled = true;
-  let email;
+  let custom = "";
```

Comment:
custom -> customClass?

Reply:
True. A lot clearer.

Reply:
Is that right? Nice, I'll check it out. Thanks.